app/mailers/discussion_mailer.rb

```diff
@@ -10,6 +10,8 @@ def parent_comment_notification( comment )
 
     subject = parent_comment_notification_subject( @reply )
 
+    ahoy_user( @parent.notification_email, @parent.author_name_any )
```

Comment:
Consider making it more explicit what work the `ahoy_user` method is doing, currently reading this here it appears a little mysterious. It might benefit from:

- possibly renaming `@user` to something generic that doesn't imply this is a `User` object. Perhaps `@recipient`, `@receiver`, or ?
- only doing the find-create work, and leave setting the `@user / @receiver` instance variable to the caller

```diff
-ahoy_user( @parent.notification_email, @parent.author_name_any )
+@receiver = find_or_create_receiver(@parent.notification_email, @parent.author_name_any)
```

```rb
def find_or_create_receiver(email_address, name = nil)
  User.find_by(email: email_address) || EmailRecipient.create_with(name: name).find_or_create_by!(email: email_address)
end
```